spec/lib/audit/catalog_to_moab_class_spec.rb

```diff
@@ -89,4 +89,38 @@
       subject
     end
   end
+
+  context '.check_version_druid' do
+    include_context 'fixture moabs in db'
+    let(:druid) { 'bj102hs9687' }
+    let(:subject) { described_class.check_version_druid(last_checked_version_b4_date, druid) }
+
+    context 'when there are PreservedCopies to check' do
+      let(:c2m_mock) { instance_double(described_class) }
+
+      it 'creates an instance and calls #check_catalog_version for every result' do
+        allow(described_class).to receive(:new).and_return(c2m_mock)
+        expect(c2m_mock).to receive(:check_catalog_version).exactly(1).times
+        described_class.check_version_druid(last_checked_version_b4_date, druid)
+      end
+
+      it 'creates an instance and calls #check_catalog_version if there are multiple druids' do
```

Comment:
uh ... I'm a little confused.  There should only be one druid, though there may be multiple preserved_copy objects ... so is this description off, or am I confused?

Assuming the former, this is the same situation as batching -- it would be useful to have some sort of counter and use expectations to show:  1.  there is more than one preserved_copy  2.  the number of times the method is called is equal to the number of preserved copies.